cmd/pmm-managed-starlark/main.go

```diff
@@ -46,4 +63,63 @@ func main() {
 	if on, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); on {
 		logrus.SetLevel(logrus.TraceLevel)
 	}
+	if on, _ := strconv.ParseBool(os.Getenv(starlarkRecursionFlag)); on {
+		resolve.AllowRecursion = true
+	}
+
+	l := logrus.WithField("component", "pmm-managed-starlark")
+
+	decoder := json.NewDecoder(os.Stdin)
+	var data checks.StarlarkScriptData
+	err := decoder.Decode(&data)
+	if err != nil {
+		l.Errorf("Error decoding json data: %s", err)
+		os.Exit(1)
+	}
+
+	results, err := runChecks(l, &data)
+	if err != nil {
+		l.Errorf("Error running starlark script: %s", err)
+		os.Exit(1)
+	}
+
+	encoder := json.NewEncoder(os.Stdout)
+	err = encoder.Encode(results)
+	if err != nil {
+		l.Errorf("error marshalling JSON results: %s", err)
+		os.Exit(1)
+	}
+}
+
+func runChecks(l *logrus.Entry, data *checks.StarlarkScriptData) ([]check.Result, error) {
+	err := unix.Setrlimit(unix.RLIMIT_CPU, &unix.Rlimit{Cur: cpuLimit, Max: cpuLimit})
+	if err != nil {
+		l.Warnf("Failed to limit CPU usage: %s", err)
```

Comment:
We should test that this line is not present in stderr (and for that we probably should move that message to constant to avoid breaking our negative test with logging tweak)